coprocess_grpc_test.go

```diff
@@ -303,6 +309,16 @@ func TestGRPCDispatch(t *testing.T) {
 		})
 	})
 
+	t.Run("Post Hook with long message", func(t *testing.T) {
```

Comment:
I think we need to add one more test with message size `>grpcTestMaxSize` to check that reply is error (not `http.StatusOK`)

Reply:
Updated tests